app/models/post.rb

```diff
@@ -1,3 +1,19 @@
 class Post < ApplicationRecord
-  belongs_to :author
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  after_create :update_user_post_counter
+
+  private
+
+  def update_user_post_counter
+    author.update(posts_count: author.posts.count)
+  end
```

Comment:
- [ ] [optional} @mariegrace31 Great work adding the suggested change but L would recommend using [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save`[Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save, this will ensure in case. where a post is created but not saved the counter isn't changed.